src/main/java/io/jlu/alfredo/commands/NatureCommand.java

```diff
@@ -0,0 +1,108 @@
+package io.jlu.alfredo.commands;
+
+import io.jlu.alfredo.utils.AlfredoUtils;
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.UnirestException;
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.MessageEmbed;
+import net.dv8tion.jda.core.events.message.MessageReceivedEvent;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class NatureCommand implements Command {
+
+    private Map<String, MessageEmbed> natureCache = new HashMap<>();
+    private final Logger LOG = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+
+    @Override
+    public void handleEvent(MessageReceivedEvent event, String parameter) {
+        String contentRaw = event.getMessage().getContentRaw().toLowerCase();
+
+        if (contentRaw.length() < "nature ".length() + 2) {
+            listAllNatures(event);
+        } else {
+            describeTargetNature(event, contentRaw);
+        }
+    }
+
+    private void listAllNatures(MessageReceivedEvent event) {
+        MessageEmbed embed = natureCache.get("");
```

Comment:
Prefer caching the data not the representation, e.g. cache the API response rather than what you make of it